EnglishTranslation-master/QUEST.tsv

```diff
@@ -3074,7 +3074,7 @@ QUEST_20151224_003081	I believe the role of Psychokinos has become even more imp
 QUEST_20151224_003082	Try as we might it's hard to tell whether the duties taken by the goddesses in the past can be eased up.{nl}
 QUEST_20151224_003083	For example if you find the legendary fantastic library, helping the goddesses inside or, should they be absent even there, restoring and maintaining it could become an enourmous task...
 QUEST_20151224_003084	Wizards aremagicians who are most devoted to fundamental magic.{nl}Isn't going back to the basics the best way to start exploring stronger types of magic?
-QUEST_20151224_003085	Using strong spells to channel strong magic is standard.{nl}But don't depend on only on one kind of magic,{nl}you must learn how to conjugate different branches of it.
+QUEST_20151224_003085	Using strong spells to channel strong magic is standard.{nl}But don't depend on only one kind of magic,{nl}you must learn how to conjugate different branches of it.
```

Comment:
`Using strong spells to channel strong magic is standard,{nl}but don't just depend on one kind of magic.{nl}You must learn how to conjugate different branches of it.`
Minor, but this works better in terms of punctuation. ("but" usually connects two sentences instead of starting one)